src/Workspaces/CSharp/Portable/Utilities/SpeculationAnalyzer.cs

```diff
@@ -508,24 +508,18 @@ private bool ReplacementBreaksCollectionInitializerAddMethod(ExpressionSyntax or
             return !SymbolsAreCompatible(originalSymbol, newSymbol);
         }
 
-        protected override bool IsInvocableExpression(SyntaxNode node)
+        protected override bool ExpressionMightReferenceMember(SyntaxNode node)
         {
-            if (node.IsKind(SyntaxKind.InvocationExpression) ||
+            return node.IsKind(SyntaxKind.InvocationExpression) ||
                 node.IsKind(SyntaxKind.ObjectCreationExpression) ||
```

Comment:
how does an object creation reference a member?  It's unclear to me what that means.  And that either means that part shouldn't be there, or hte name of the containing method is a bit misleading .  Can you help clarify?

Reply:
It was part of the earlier `IsInvocableExpression`, which isn't good justification to keep it around. Removed it and ran a smoke test locally and no inlining or remove cast tests blew up. Going to let the full test suite run although I believe you are right and it is superfluous. Thanks